src/include/connection.h

```diff
@@ -246,6 +246,14 @@ struct __wt_connection_impl {
     uint64_t hash_size;       /* General hash bucket array size */
     int is_new;               /* Connection created database */
 
+    uint32_t recovery_major; /* Database recovery major version */
+    uint32_t recovery_minor; /* Database recovery minor version */
+    uint32_t recovery_patch; /* Database recovery patch version */
+
+#ifndef WT_STANDALONE_BUILD
+    bool unclean_shutdown; /* Flag to indicate the earlier shutdown status */
```

Comment:
Maybe it's better to initialize it with `false` explicitly on connection open?

Reply:
By default it is false. Better to avoid any problems, explicitly set it to false.